packages/template-retail-react-app/package.json

```diff
@@ -79,11 +81,11 @@
     "bundlesize": [
         {
             "path": "build/main.js",
-            "maxSize": "52 kB"
+            "maxSize": "58 kB"
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "270 kB"
+            "maxSize": "338 kB"
```

Comment:
We might want to reconsider the bundle sizes actively when we are ready to merge this main feature.

Reply:
Note: this is why the bundle has ballooned in size. The commerce-sdk-isomorphic package is duplicated.

![Safari 2022-10-28 at 12 26 09](https://user-images.githubusercontent.com/847300/199278176-ee93de6c-a74e-46a7-ae01-7a14421c5497.png)